turbo.json

```diff
@@ -163,7 +163,7 @@
     },
     "test:unit": {
       "inputs": [
-        "{app,components,hooks,i18n,layouts,middlewares,pages,providers,types,util}/**/*.{ts,tsx}",
+        "{app,components,hooks,i18n,layouts,middlewares,pages,providers,types,util}/**/*.{ts,tsx,mjs}",
```

Comment:
@ovflowd I think I did this correctly, would appreciate a second set of eyes. without this, turbo kept thinking the tests were unchanged